installation.md

```diff
@@ -135,7 +135,7 @@ From there, we can build and run Nu if we are using `rustup` with:
 
 ```
 > cd nushell
-nushell> cargo build --all --features=stable && cargo run --all --features=stable
+nushell> cargo build --all-features && cargo run --all-features
```

Comment:
This won't work correctly. Basically `--all-features` also adds the tracer which we need to fix. For `cargo build` and `cargo run` we better leave them as `--all --features=stable`

Reply:
--all is not supported by `cargo run` as far as I know

Reply:
omg, you are right. Then we may want to have `cargo run --features=stable` (not `--all-features`)

Reply:
For the case of `cargo build` I also agree with @jonathandturner 's comment.

Reply:
Ok, updated to use @jonathandturner 's suggestion for `cargo build` and yours for `cargo run`